src/libcore/mem/mod.rs

```diff
@@ -358,12 +358,13 @@ pub fn size_of_val<T: ?Sized>(val: &T) -> usize {
 ///     - an (unstable) [extern type], then this function is always safe to
 ///       call, but may panic or otherwise return the wrong value, as the
 ///       extern type's layout is not known. This is the same behavior as
-///       [`size_of_val`] on a reference to an extern type tail.
+///       [`size_of_val`] on a reference to a type with extern type tail.
 ///     - otherwise, it is conservatively not allowed to call this function.
 ///
 /// [slice]: ../../std/primitive.slice.html
 /// [trait object]: ../../book/ch17-02-trait-objects.html
 /// [extern type]: ../../unstable-book/language-features/extern-types.html
+/// [`size_of_val`]: ../../core/mem/fn.size_of_val.html
```

Comment:
Is this needed? https://doc.rust-lang.org/nightly/core/mem/fn.size_of_val.html already correctly links to `size_of_val`

Reply:
This is in `size_of_val_raw` though? I do not understand the question.

Reply:
I meant that https://doc.rust-lang.org/nightly/core/mem/fn.size_of_val_raw.html correctly links the `size_of_val` :sweat_smile:

Reply:
Oh... that's weird, I just saw it with a broken link somewhere... let me see if I can find that again.

Reply:
Ah, the link is broken here:
https://doc.rust-lang.org/nightly/std/mem/fn.size_of_val_raw.html
Sounds like a rustdoc bug then? Cc @rust-lang/rustdoc

Reply:
It's #65983. The intra-doc link feature isn't ready for use in `std` yet.

Reply:
Okay, seems like my patch makes sense then as it avoids using that feature.